bin/reload

```diff
@@ -2,7 +2,7 @@
 
 require __DIR__ . '/base_script.php';
 
-show_run("Install dependencies", "composer install");
```

Comment:
because, I start reload and in new generate paramater.yml and then disappears:
admin_dashboard_base_url: /admin
    sonata_admin.title: 'Dream Project'
    sonata_admin.logo_title: bundles/app/img/dream.png